pulp_container/app/migrations/0013_add_delete_versions_perm.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.17 on 2021-01-28 19:31
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('container', '0012_add_container_namespace_permissions'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='containerrepository',
+            options={'default_related_name': '%(app_label)s_%(model_name)s', 'permissions': [('sync_containerrepository', 'Can start a sync task'), ('modify_content_containerrepository', 'Can modify content in a repository'), ('build_image_containerrepository', 'Can use the image builder in a repository'), ('delete_containerrepository_versions', 'Can delete repository versions')]},
```

Comment:
I need to rebase, waiting for other PRs with migrations to be merged since mine is small.